modules/biobank/php/container.class.inc

```diff
@@ -0,0 +1,350 @@
+<?php declare(strict_types=1);
+/**
+ * Container Object Class
+ * This class contains an instance of a database handling that is needed to
+ * permanently store and retrieve Container Object instances.
+ *
+ * PHP Version 7.2
+ *
+ * @category   Entity
+ * @package    Loris
+ * @subpackage Biobank
+ * @author     Henri Rabalais <henri.rabalais@mcin.ca>
+ * @license    http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link       https://www.github.com/aces/Loris/
+ */
+
+namespace LORIS\biobank;
+
+/**
+ * Container Object Class
+ * This class contains an instance of a database handling that is needed to
+ * permanently store and retrieve Container Object instances.
+ *
+ * PHP Version 7.2
+ *
+ * @category Loris
+ * @package  Biobank
+ * @author   Henri Rabalais <hrabalais.mcin@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+class Container implements \JsonSerializable, \LORIS\Data\DataInstance
+{
+    /**
+     * Persistent Instance variables.
+     *
+     * These properties describe a container entity.
+     *
+     * @var int   $id 
+     * @var int   $barcode
+     * @var int   $typeId
```

Comment:
Could you maybe use this annotation as an opportunity to clarify the purpose of $typeId as you explained elsewhere in this PR? The distinction might get lost otherwise.

Reply:
Yeah, but I'm really not sure how to elucidate that in a var documentation comment. 

`@var int $typeId The ID of the container type` seems a bit redundant given my setters and getters.